en/config_rover/differential.md

```diff
@@ -0,0 +1,305 @@
+# Configuration/Tuning (Differential Rover)
+
+This topic provides a step-by-step guide for setting up your [Differential rover](../frames_rover/differential.md).
+Successive steps enable [drive modes](../flight_modes_rover/differential.md) with more autopilot support and features.
+
+::: warning
+Each step is dependent on the previous steps having been completed.
+Modes will only work properly if the preceding modes have been configured.
+:::
+
+## Basic Setup
+
+To start using the differential rover:
+
+1. Enable Rover support by flashing the [PX4 rover build](../frames_rover/index.md#flashing-the-rover-build) onto your flight controller.
+   Note that this is a special build that contains rover-specific modules.
+
+2. In the [Airframe](../config/airframe.md) configuration select _Generic Rover Differential_ frame:
+
+   ![QGC screenshot showing selection of the airframe 'Generic Rover Differential'](../../assets/config/airframe/airframe_generic_rover_differential.png)
+
+   Select the **Apply and Restart** button.
+
+   ::: info
+   If this airframe does not show up in the UI, it can alternatively be selected by setting the [SYS_AUTOSTART](../advanced_config/parameter_reference.md#SYS_AUTOSTART) parameter to `50000`.
+   :::
+
+3. Use [Actuators Configuration & Testing](../config/actuators.md) to map the motor functions to flight controller outputs.
+
+## Manual Mode
+
```

Comment:
We might have one sentence to explain the mode, but now that docs are ordered to describe modes first I think we can assume that the user knows the differences.